internal/config/canary_config_test.go

```diff
@@ -45,7 +47,7 @@ func TestConfigDefault(t *testing.T) {
 }
 
 func TestConfigCustom(t *testing.T) {
-	os.Setenv(BootstrapServersEnvVar, "my-cluster-kafka:9092")
+	os.Setenv(BootstrapServersEnvVar, "my-cluster-kafka:9092,my-cluster-kafka:9093")
```

Comment:
This does not seem to make much sense in Strimzi context. Would it be better to have some value like `broker1:9092,broker2:9092`?